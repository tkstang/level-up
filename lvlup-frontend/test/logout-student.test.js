import LogoutStudent from '../src/components/Navbar/logout-student';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('LogoutStudent should render', () => {
  const store = mockStore({ lvlupApp });
  const logout = shallow(
    <LogoutStudent store={store} />,
  );
  expect(shallowToJson(logout)).toMatchSnapshot();
});
