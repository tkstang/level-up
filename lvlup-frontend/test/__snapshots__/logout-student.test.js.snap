exports[`test LogoutStudent should render 1`] = `
<Link
  replace={false}
  to="/">
  <Button
    as="button"
    color="orange"
    inverted={true}
    onClick={[Function]}
    size="small">
    <Icon
      as="i"
      name="github" />
     Log Out
  </Button>
</Link>
`;
